amclient/amclientargs.py

```diff
@@ -52,6 +52,11 @@
     help='Purpose of storage space location valid choices: "AR", "AS", "CP", "DS", "SD", "SS", "BL", "TS", "RP"',
     type=None,
 )
+EVENT_REASON = Arg(
+    name="event_reason", help="Event reason for package deletion request", type=None
+)
+SS_USER_ID = Arg(name="ss_user_id", help="Storage Service user ID", type=None)
+SS_USER_EMAIL = Arg(name="ss_user_email", help="Storage Service user email", type=None)
```

Comment:
It might be worth to also document these arguments in: https://github.com/artefactual-labs/amclient/blob/5ea6fa0346a26c480b94ad4a6bcea20ffb9d3939/amclient/amclient.py#L108-L123